src/policy/descriptor.rs

```diff
@@ -12,6 +11,16 @@ use elements_miniscript::{MiniscriptKey, ToPublicKey};
 use std::fmt;
 use std::str::FromStr;
 
+pub trait UnspendableKey: MiniscriptKey {
+    fn unspendable() -> Self;
+}
```

Comment:
ad6f865b868b7add369d26c5aa1a6195b7e8b703: The name `UnspendableKey` can lead someone to believe that all keys of this type are unspendable. Maybe `HasUnspendable` is less confusing.